DataImport.E2E.Tests/features/import.feature

```diff
@@ -25,6 +25,7 @@ Feature: Import
   Scenario: Import Data
     When selecting a valid file to import
     And clicking import
+    And select api version
```

Comment:
@jpardogrowthaccelerationpartners 
Why we select api version ?

Reply:
because the api version is not automatically selected

Reply:
step remove. thanks